src/Service.Host/client/src/components/PurchaseOrders/CreatePurchaseOrderUt.js

```diff
@@ -454,6 +463,14 @@ function CreatePurchaseOrderUt() {
                                         }))}
                                     onChange={(propertyName, selected) => {
                                         handleSupplierChange({ id: selected });
+                                        handleDetailValueFieldChange(
```

Comment:
I would change this to be the same dispatch as the one above for consistency. (i.e. dispatch an action of type detailFieldUpdate in both cases)

Reply:
This is one of the functions that has some currency conversion before the actual dispatch. I think for uniformity sake, it may stick out if I change it? There's another instance of HandleDetailValueFieldChange within the code as well.